dart/lib/src/sentry_envelope_item_header.dart

```diff
@@ -19,6 +22,7 @@ class SentryEnvelopeItemHeader {
   /// Item header encoded as JSON
   Future<Map<String, dynamic>> toJson(int length) async {
     return {
+      if (itemCount != null) 'item_count': itemCount,
```

Comment:
Why does the [logs header] hav to have `item_count`, when we are already sending `length`? 🤔